src/components/DeploymentQuickRunOverflowMenuItem.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <p-overflow-menu-item @click="run">
+    <slot>
+      Quick run
+    </slot>
+  </p-overflow-menu-item>
+</template>
+
+<script lang="ts" setup>
+  import { showToast } from '@prefecthq/prefect-design'
+  import { h } from 'vue'
+  import { useRouter } from 'vue-router'
+  import ToastFlowRunCreate from './ToastFlowRunCreate.vue'
+  import { useWorkspaceApi, useWorkspaceRoutes } from '@/compositions'
+  import { localization } from '@/localization'
+
+  const props = defineProps<{
+    deploymentId: string,
+  }>()
+
+  const api = useWorkspaceApi()
+  const router = useRouter()
+  const routes = useWorkspaceRoutes()
+
+  const run = async (): Promise<void> => {
+    try {
+      const flowRun = await api.deployments.createDeploymentFlowRun(props.deploymentId, {
+        state: {
+          type: 'scheduled',
+          message: 'Run from the Prefect UI with defaults',
+        },
+      })
+
+      const toastMessage = h(ToastFlowRunCreate, { flowRun, flowRunRoute: routes.flowRun, router, immediate: true })
+      showToast(toastMessage, 'success')
+    } catch (error) {
+      showToast(localization.error.scheduleFlowRun, 'error')
+      console.warn(error)
```

Comment:
We should always log errors as errors to the console. Warnings should be reserved for something that didn't fail per say but might not be what the developer intended. 

```suggestion
      console.error(error)
```